src/ring_theory/valuation/basic.lean

```diff
@@ -293,6 +293,13 @@ begin
   simpa only [v.map_one, v.map_neg] using v.map_add_eq_of_lt_left h
 end
 
+lemma one_lt_val_iff
+  {K : Type*} [division_ring K] (v : valuation K Γ₀) {x : K} (h : x ≠ 0) :
+  1 < v x ↔ v x⁻¹ < 1 :=
+begin
+  simpa using (inv_lt_inv₀ (v.ne_zero_iff.2 h) one_ne_zero).symm,
+end
+
```

Comment:
Can you move `K` to the variables line at the top of the file?

Reply:
Also a small golf:
```suggestion
lemma one_lt_val_iff [division_ring K] (v : valuation K Γ₀) {x : K} (h : x ≠ 0) :
  1 < v x ↔ v x⁻¹ < 1 :=
by simpa using (inv_lt_inv₀ (v.ne_zero_iff.2 h) one_ne_zero).symm,

```

Reply:
If I move `K` to the variables line then would it be appropriate to remove every instance of `{K : Type*}` throughout the file? I only ask because it seems like something that would've been done already if there weren't a reason not to

Reply:
Usually it's not done already simply because it's more work. That sounds good to me though.